stable/chaoskube/values.yaml

```diff
@@ -40,6 +40,9 @@ excludedDaysOfYear:
 # Set specific Timezone for Actions to take place
 timezone: UTC
 
+# If nothing set, defaults to "text". Switch to "json" to enable structured logging
+logFormat:
```

Comment:
You could use `logFormat: text` here and leave out the `{{- if .Values.logFormat }}` stuff in `deployment.yaml`.

I tend to like that more in this chart but ultimately you can decide 😄

Reply:
That's what I did initially before I found out `logFormat`: `empty` was failing. Happy to revert this and leave text in the `helm values`. ☺️

Reply:
For upgrading it's actually better to do the `if` stuff. Thank you 😃